app/locales/ru.json

```diff
@@ -1,193 +1,193 @@
     "pause": "Пауза",
-    "forHour": "час",
-    "for2Hours": "2 часа",
-    "for5Hours": "5 часов",
+    "forHour": "на час",
+    "for2Hours": "на 2 часа",
+    "for5Hours": "на 5 часов",
     "untilMorning": "до утра",
     "indefinitely": "на неопределённый срок",
     "resetBreaks": "Сбросить перерывы",
     "quitStretchly": "Выход из Stretchly",
-    "restoreDefaults": "Restore defaults",
-    "warning": "This will restore app defaults and all your settings will be lost.",
-    "continue": "Continue",
-    "cancel": "Cancel",
+    "restoreDefaults": "Восстановить настройки по умолчанию",
+    "warning": "Это действие вернет настройки к первоначальным, ваши настройки будут утеряны.",
+    "continue": "Продолжить",
+    "cancel": "Отменить",
     "preferences": "Настройки"
   },
   "break": {
-    "postpone": "Postpone this break",
-    "skip": "Skip this break",
+    "postpone": "Отложить",
+    "skip": "Пропустить",
     "title": "Перерыв!"
   },
   "preferences": {
-    "title": "Stretchly Preferences",
+    "title": "Настройки Stretchly",
     "nav": {
-      "settings": "Settings",
-      "schedule": "Schedule",
-      "theme": "Theme",
-      "about": "About",
+      "settings": "Настройки",
+      "schedule": "Расписание",
+      "theme": "Тема",
+      "about": "О программе",
       "heart": "Love Stretchly"
     },
     "settings": {
-      "openAtLogin": "Start Stretchly automatically when loggin in",
-      "showBreaksIn": "Shows breaks in:",
-      "window": "Window",
-      "fullscreen": "Full screen",
-      "showIdeas": "Show exercise tips during breaks",
-      "allScreens": "Shows breaks on all monitors",
-      "monitorIdleTime": "Monitor system idle time (countdown is paused if system is idle)",
-      "monitorDnd": "Show breaks even in Do Not Disturb mode",
-      "language": "Select langauge:",
-      "restoreDefaults": "Restore defaults"
+      "openAtLogin": "Запускать Stretchly автоматически при входе в систему",
+      "showBreaksIn": "Показывать перерывы:",
+      "window": "в окне",
+      "fullscreen": "на весь экран",
+      "showIdeas": "Показывать идеи для упражнений во время перерывов",
+      "allScreens": "Показывать перерывы на всех экранах",
+      "monitorIdleTime": "Следить за тем, когда система не используется (отсчёт времени до перерывов останавливается)",
+      "monitorDnd": "Показывать перерывы даже в режиме \"Не беспокоить\"",
+      "language": "Выбрать язык:",
+      "restoreDefaults": "Восстановить настройки по умолчанию"
     },
     "schedule": {
-      "miniBreaks": "Mini Breaks:",
-      "miniBreaksInfo": "Mini Breaks are short breaks taken regularly to give you a chance to stretch and relax.",
-      "enableMiniBreaks": "Enable Mini Breaks",
+      "miniBreaks": "Мини-перерывы:",
+      "miniBreaksInfo": "Мини-перерывы это регулярные короткие перерывы, чтоб размяться и отдохнуть.",
+      "enableMiniBreaks": "Включить мини-перерывы",
       "breakFor": "Break for:",
```

Comment:
What is meant here and in the next one?

Reply:
Here's the pic, it's the settings for Mini breaks

![image](https://user-images.githubusercontent.com/4745859/83414146-48c37f80-a41d-11ea-8eb7-972f9ac1906e.png)